inc/admin/dashboard/namespace.php

```diff
@@ -379,10 +369,29 @@ function display_pressbooks_blog() {
 
 		set_site_transient( 'pb_rss_widget', $rss, DAY_IN_SECONDS );
 	}
-
+	// @codingStandardsIgnoreLine
```

Comment:
Don't know how to properly escape this output, so I simply exempted it from coding standards.